datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mappers/DatasetMapper.java

```diff
@@ -49,6 +55,18 @@ public Dataset apply(@Nonnull final com.linkedin.dataset.Dataset dataset) {
         if (dataset.hasOwnership()) {
             result.setOwnership(OwnershipMapper.map(dataset.getOwnership()));
         }
+        if (dataset.hasDeprecation()) {
+            result.setDeprecation(DatasetDeprecationMapper.map(dataset.getDeprecation()));
+        }
+        if (dataset.hasInstitutionalMemory()) {
```

Comment:
where is this method defined?

Reply:
This method is part of the `Dataset` auto-generated Rest.li object by default

Reply:
Ah, thank you!